scripts/install-windows-prereqs.ps1

```diff
@@ -35,7 +35,7 @@ Param(
     [string]$GetPipURL = 'https://bootstrap.pypa.io/3.4/get-pip.py',
     [string]$GetPipHash = '564FABC2FBABD9085A71F4A5E43DBF06D5CCEA9AB833E260F30EE38E8CE63A69',
     [Parameter(mandatory=$true)][string]$InstallPath,
-    [Parameter(mandatory=$true)][ValidateSet("SGX1FLC", "SGX1", "SGX1FLC-NoDriver", "SGX1-NoDriver")][string]$LaunchConfiguration,
+    [Parameter(mandatory=$true)][ValidateSet("SGX1FLC", "SGX1", "SGX1FLC-NoPSWDCAP", "SGX1-NoPSWDCAP")][string]$LaunchConfiguration,
```

Comment:
Would something like `-NoIntelDrivers` be a better name? I initially thought `-NoPSWDCAP` meant installing DCAP but not PSW.

Reply:
I have rebuilt the images to reference this flag, but this is a good point. Would you accept merging this in and then running a find/replace as a follow-up? If I change the flag now.. we delay the release by another day which is where my concern lies?

Reply:
Works for me

Reply:
@rs-- I ended up needing to wait on automation, I pushed out the prod images with the underlying commit being before the refactor. I went ahead and added your changes requested. If e2e fails, I will remove the change and merge without. If it passes, merge with.